examples/transparent_layer_bar.json

```diff
@@ -0,0 +1,20 @@
+{
+  "data": { "url": "data/population.json",
+    "filter": "datum.year == 2000",
+    "calculate": [{"field": "Gender", "expr": "datum.sex == 2 ? \"Female\" : \"Male\""}]
+  },
+  "mark": "bar",
+  "encoding": {
+    "y": {"field": "people","type": "quantitative", "aggregate": "sum"},
+    "x": {"field": "age", "type": "ordinal"},
+    "color": {"field": "Gender","type": "nominal",
+      "scale": {"range": ["red","blue"]}
```

Comment:
I think this is easier for people to understand the ratio because females are often associated with red-ish colors, males are with blue-ish colors.